### PR TITLE
N2WS dashboards improvements

### PR DESCRIPTION
**Pull request by eliadeini**
### What does this PR do?

Updates N2WS dashboards

### Motivation

N2WS Backup & Recovery version 3.2.1 came to market. New features were added + bug fixes

### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests (Manual testing)
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?
Improvements that were added:
1) Alerts (event) view improved and now are filterable
2) % was added to DR backups
3) Entity dashboard was cleaned from irrelevant DR data
4) Data about fSx snapshots was added

### CODE REVIEW

File: n2ws/manifest.json [hunk 1]
```diff
@@ -26,9 +26,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
-	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json"
+	  "N2WSBackup&Recovery-EntityTypesDetails": "assets/dashboards/N2WSBackup&Recovery-EntityTypesDetails.json",
```
Comment (hithwen) [inline n2ws/manifest.json:29]: The keys in this json cannot be changed once they have been submitted. It was already mentioned on your previous PR https://github.com/DataDog/integrations-extras/pull/760#discussion_r525258661

Comment (eliadeini) [reply n2ws/manifest.json:29]: I didn't touch the menifest.json, these changes probably came from git pull.
I'll revert this file.

Comment (hithwen) [reply n2ws/manifest.json]: It looks like you have renamed all dashboard json files, is that expected? If you do so you need to rename the values in these key-value pairs.

Comment (eliadeini) [reply n2ws/manifest.json]: No, it is not expected. You gave me a comment about this issue, so I reverted the names back to what it was before.
I copied the current names from your repository on Github in order to make sure I'm not changing anything in the manifest.json in this Pull Request.

Comment (hithwen) [reply n2ws/manifest.json]: You cannot change the keys in these pairs but if you change the filenames then you need to change the values

File: n2ws/manifest.json [hunk 2]
```diff
@@ -26,9 +26,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
```
Comment (hithwen) [inline n2ws/manifest.json:29]: It seems assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json has been removed as part of this PR

File: n2ws/manifest.json (hunk 1 above)
Comment (eliadeini) [reply n2ws/manifest.json]: I didn't change the filenames. Probably an old file from my repository accidently entered my first commit when I merged, I reverted the change.
The only thing that should be changed due to this merge request are the dashboards JSON files.

**Review by hithwen — CHANGES_REQUESTED** [review]
image
It seems there are multiple unintended file changes on this PR. I suggest you update this branch with the latest master to help clear them up

**Comment by eliadeini** [comment]
I'll make a new one with a clean branch

**Comment by eliadeini** [comment]
Created a new Pull Request instead: https://github.com/DataDog/integrations-extras/pull/808

Please review

**Pull request closed without merging**